composer.json

```diff
@@ -10,14 +10,13 @@
         }
     ],
     "require": {
-        "ezsystems/ezplatform-kernel": "^1.0",
+        "ezsystems/ezplatform-kernel": "~1.0",
```

Comment:
Why this change? Using `^1.0` and `~1.0` amounts to the same thing, but `^` is more semantic.

Reply:
I've reverted this.  I was having issues with composer pulling in the wrong deps.